example.py

```diff
@@ -0,0 +1,7 @@
+from dbnd import task
```

Comment:
why do we need this file at root folder? 
we have whole "examples" folder

Reply:
my bad - I played around with commandline and forgot to remove this file